spacy/ml/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .entity_linker import *
+from .parser import *
+from .tagger import *
+from .tensorizer import *
+from .textcat import *
+from .tok2vec import *
```

Comment:
This sucks, but I put it in as a quick hack to make sure the `@registry` annotations were properly registered in those files. Needs a proper solution.